### ISSUE TITLE
How to access environment or Key values from a 'active build preset'

### ISSUE THREAD

**Issue by harshakm-50063** [MicrosoftDocs/cpp-docs#3717]
[Enter feedback here]
While the document seems to talk about 'active configuration preset' it does not say how to access it from 'active build preset'. Any details on this will help.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 18671a37-b8d9-b8cd-51fc-c1cd267e6af5
* Version Independent ID: 6f0932ad-998a-9449-dfff-259452ba20d5
* Content: [Configure and build with CMake Presets](https://docs.microsoft.com/en-us/cpp/build/cmake-presets-vs?view=msvc-170#variable-substitution-in--launchvsjson-and--tasksvsjson)
* Content Source: [docs/build/cmake-presets-vs.md](https://github.com/Microsoft/cpp-docs/blob/master/docs/build/cmake-presets-vs.md)
* Product: **visual-cpp**
* Technology: **cpp-tools**
* GitHub Login: @corob-msft
* Microsoft Alias: **corob**

**Comment by corob-msft** [comment]
@harshakm-50063 

I'm sorry you're not finding the information you need. I'd expect a [build preset](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#build-preset) to include a configuration preset. It ought to provide access to the same environment values as the configuration preset. But perhaps I don't understand your question.

**Comment by harshakm-50063** [comment]
> @harshakm-50063
> 
> I'm sorry you're not finding the information you need. I'd expect a [build preset](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#build-preset) to include a configuration preset. It ought to provide access to the same environment values as the configuration preset. But perhaps I don't understand your question.

Yes. Build Preset, does includes a configuration preset. I can access environment variables defined in a 'configuration preset' using a ${env.Variable} as well as cmake preset tags as well using '$cmake.key}. 
However, this works for the 'active configuration preset' and does NOT work for 'active build preset'
